init.py

```diff
@@ -110,32 +110,22 @@ def execute_lctk(argv):
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{rbsa_pipeline.run_dir}/total_loadshapes/', 'total_loadshapes.png')
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{rbsa_pipeline.run_dir}/loadshapes/', 'loadshapes.png')
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{rbsa_pipeline.run_dir}/components/', 'components.png')
+    image_stitcher.stitch(f'{base.LOCAL_PATH}/{rbsa_pipeline.run_dir}/normal_loadshapes/', 'normalized_loadshapes.png')     
+    image_stitcher.stitch(f'{base.LOCAL_PATH}/{rbsa_pipeline.run_dir}/enduse_loadshapes/', 'enduse_loadshapes.png')        
+    image_stitcher.stitch(f'{base.LOCAL_PATH}/{rbsa_pipeline.run_dir}/total_loadshapes/', 'total_loadshapes.png')      
+    image_stitcher.stitch(f'{base.LOCAL_PATH}/{rbsa_pipeline.run_dir}/loadshapes/', 'loadshapes.png')      
+    image_stitcher.stitch(f'{base.LOCAL_PATH}/{rbsa_pipeline.run_dir}/components/', 'components.png')      
 
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{ceus_pipeline.run_dir}/ceus_normal_loadshapes/', 'ceus_normal_loadshapes.png')
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{ceus_pipeline.run_dir}/ceus_enduse_loadshapes/', 'ceus_enduse_loadshapes.png')
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{ceus_pipeline.run_dir}/ceus_total_loadshapes/', 'ceus_total_loadshapes.png')
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{ceus_pipeline.run_dir}/ceus_loadshapes/', 'ceus_loadshapes.png')
+    image_stitcher.stitch(f'{base.LOCAL_PATH}/{ceus_pipeline.run_dir}/ceus_normal_loadshapes/', 'ceus_normal_loadshapes.png')     
+    image_stitcher.stitch(f'{base.LOCAL_PATH}/{ceus_pipeline.run_dir}/ceus_enduse_loadshapes/', 'ceus_enduse_loadshapes.png')      
+    image_stitcher.stitch(f'{base.LOCAL_PATH}/{ceus_pipeline.run_dir}/ceus_total_loadshapes/', 'ceus_total_loadshapes.png')        
+    image_stitcher.stitch(f'{base.LOCAL_PATH}/{ceus_pipeline.run_dir}/ceus_loadshapes/', 'ceus_loadshapes.png')        
     image_stitcher.stitch(f'{base.LOCAL_PATH}/{ceus_pipeline.run_dir}/ceus_components/', 'ceus_components.png')
 
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{mix_pipeline.run_dir}/suburban_mix/', 'suburban_mix.png')
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{mix_pipeline.run_dir}/urban_mix/', 'urban_mix.png')
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{mix_pipeline.run_dir}/mixed_mix/', 'mixed_mix.png')
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{mix_pipeline.run_dir}/rural_mix/', 'rural_mix.png')
-
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{mix_pipeline.run_dir}/suburban_mix_hour_norm/', 'suburban_mix_hour_norm.png')
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{mix_pipeline.run_dir}/urban_mix_hour_norm/', 'urban_mix_hour_norm.png')
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{mix_pipeline.run_dir}/mixed_mix_hour_norm/', 'mixed_mix_hour_norm.png')
-    image_stitcher.stitch(f'{base.LOCAL_PATH}/{mix_pipeline.run_dir}/rural_mix_hour_norm/', 'rural_mix_hour_norm.png')
```

Comment:
What's the reason for adding the image stitching back in?

Reply:
Was testing something and must have left it in, deleted now